### ISSUE TITLE
[SN2017egm] <Repeated aliases>

### ISSUE THREAD

**Issue by mnicholl** [astrocatalogs/supernovae#111]
Aliases look pretty messed up -- says "SN2017egm" like 10 times with the same reference. I guess because of model uploads?

**Comment by guillochon** [comment]
Yep I've seen this bug, thanks for making an issue for it. As you suggest it's from each of the models, the catalog isn't quite reading those in correctly.
